loom/graphics/gfxVectorRenderer.cpp

```diff
@@ -353,10 +353,18 @@ void VectorRenderer::destroyGraphicsResources()
 
 void VectorRenderer::initializeGraphicsResources()
 {
+    int flags = 0;
+
+#if GFX_OPENGL_CHECK
+    flags |= NVG_DEBUG;
+#endif
+
+    flags |= NVG_ANTIALIAS | NVG_STENCIL_STROKES;

```

Comment:
Maybe just Stage.vectorQuality with bit 0 being AA on and bit 1 being
stencil on?

On Mon, May 4, 2015 at 2:42 PM, Miha Lunar <notifications@github.com> wrote:

> In loom/graphics/gfxVectorRenderer.cpp
> <https://github.com/LoomSDK/LoomSDK/pull/311#discussion_r29626873>:
>
> > @@ -353,10 +353,18 @@ void VectorRenderer::destroyGraphicsResources()
> >
> >  void VectorRenderer::initializeGraphicsResources()
> >  {
> > +    int flags = 0;
> > +
> > +#if GFX_OPENGL_CHECK
> > +    flags |= NVG_DEBUG;
> > +#endif
> > +
> > +    flags |= NVG_ANTIALIAS | NVG_STENCIL_STROKES;
>
> That was my idea as well, any API suggestions? I was thinking the
> ANTIALIAS one should be some nanovg/vector specific switch as we may want
> to support actual™ OpenGL multisample antialiasing later on.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LoomSDK/LoomSDK/pull/311/files#r29626873>.
>